backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/starrating/BpkStarRating.kt

```diff
@@ -107,10 +134,23 @@ private fun BpkStarRating(
   Row(modifier = modifier.semantics { this.contentDescription = scope.contentDescription(roundedRating, maxRating) }) {
     for (item in 0 until maxRating) {
       val value = (roundedRating - item).coerceIn(0f, 1f)
+      val starModifier = Modifier.clickable { onRatingSelected?.invoke(item + 1) }
```

Comment:
this will always apply the `clickable` logic to the modifier (and therefore make it seems as if it's clickable) even when the callback is null - you can use the `applyIf` function to only apply it if the callback is not null